### ISSUE TITLE
Behavior to support commenting behavior in publiclab.org

### ISSUE THREAD

**Issue by ananyo2012** [publiclab/plotsbot#71]
Now that commenting via token has been done in plots2 we should add a behavior that would listen to a mention for `plotsbot` in the comment and respond with a reply. We need to make a POST request with the token header.

**Comment by jywarren** [comment]
+1!

**Comment by ryzokuken** [comment]
@ananyo2012 Right now, the commenting process in one-way. The bot can make comments on the website, but it has no way of knowing if someone else made a comment. Perhaps we could add such a functionality?

**Comment by jywarren** [comment]
Ah, so something like:

1. see recent posts
2. identify a post and leave a comment on it
3. query the site for what comments are on that post (requires new API call)
4. reply again in a new comment

so we'd need 3), right?

**Comment by jywarren** [comment]
Could that look like:

`https://publiclab.org/api/comments/node/<nid>/` ?

**Comment by ryzokuken** [comment]
@jywarren once we have the persistent behavior trigger, adding to the API could be done easily.

One of us could make an issue for that on the plots2 repository. (Or is it too early?)

**Comment by jywarren** [comment]
yes, please do open an issue for that! Does the URL look reasonable?

**Comment by ryzokuken** [comment]
@jywarren It does! I will make an issue for that in a moment.
